### DISCUSSION TITLE
Stop displaying module version in diagnostic

### DISCUSSION THREAD

**Discussion by hasithaa** [ballerina-platform/ballerina-lang#28720] [Ideas]
This is noticed when testing the SwanLake Alpha2 pack. 

Eg: 
```
ERROR [tmp.bal:(4:16,4:31)] incompatible types: expected 'string', found '(ballerina/file:0.7.0-alpha3:InvalidOperationError|ballerina/file:0.7.0-alpha3:PermissionError|ballerina/file:0.7.0-alpha3:FileSystemError|ballerina/file:0.7.0-alpha3:FileNotFoundError|ballerina/file:0.7.0-alpha3:NotLinkError|ballerina/file:0.7.0-alpha3:IOError|ballerina/file:0.7.0-alpha3:SecurityError|ballerina/file:0.7.0-alpha3:InvalidPathError|ballerina/file:0.7.0-alpha3:InvalidPatternError|ballerina/file:0.7.0-alpha3:RelativePathError|ballerina/file:0.7.0-alpha3:UNCPathError|ballerina/file:0.7.0-alpha3:GenericError)?'
error: compilation contains errors
```

Since we are not supporting multiple versions of the same module, we can simply omit the version part from the diagnostics. This will improve the readability. 

@sameerajayasoma @hevayo @pubudu91 @dulajdilshan @nadeeshaan @rasika Thoughts?

**Comment by hasithaa** [comment]
We decided to remove the version part of the diagnostic.

**Reply by rasika** [reply]
+1 to remove versions from diagnostics. Still we'll be getting below due to https://github.com/ballerina-platform/ballerina-lang/issues/11056 
```
ballerina/file:InvalidOperationError|ballerina/file:PermissionError|ballerina/file:FileSystemError|ballerina/file:FileNotFoundError|ballerina/file:NotLinkError|ballerina/file:IOError|ballerina/file:SecurityError|ballerina/file:InvalidPathError|ballerina/file:InvalidPatternError|ballerina/file:RelativePathError|ballerina/file:UNCPathError|ballerina/file:GenericError
``` 

where it **should** be actually `file:Error`

**Comment by nadeeshaan** [comment]
I would say we remove the orgname part as well. In a related note, what will happen if we have an alias?

**Reply by hasithaa** [reply]
We can have the same module name in two different organizations, hence we can't remove it.

**Reply by nadeeshaan** [reply]
We are experiencing the same issue when representing the information in VSCode as well. Please refer the bellow,

<img width="1069" alt="Screenshot 2021-02-19 at 08 03 46" src="https://user-images.githubusercontent.com/1329674/108451061-2cf25900-728c-11eb-9ed6-b591d9b7be45.png">
